lib/shared_data/include/Utility.h

```diff
@@ -18,6 +19,24 @@ struct veh_vec
     T FR;
     T RL;
     T RR;
+
+    void construct(T _FL, T _FR, T _RL, T _RR)
```

Comment:
make this just the constructor, no need to have a function for this and delete the default constructor